<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSessionManage</class>
 <widget class="QWidget" name="FormSessionManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>场次编辑</string>
  </property>
  <widget class="QComboBox" name="filmComboBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="cinemaComboBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="filmLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置电影</string>
   </property>
  </widget>
  <widget class="QLabel" name="ScreenLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置放映厅</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>640</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>640</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="showtimeLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置放映日期,时间(在8:00-22:00之间)</string>
   </property>
  </widget>
  <widget class="QLabel" name="yuanLabel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>550</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>元</string>
   </property>
  </widget>
  <widget class="QLabel" name="priceLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>550</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置价格</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="PriceSpinBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>550</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>344</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>290</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="screenComboBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="CinemaLabel">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>100</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>影院</string>
   </property>
  </widget>
  <widget class="QLabel" name="ScreenLabel_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>170</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>放映厅</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
